src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -150,10 +155,37 @@ const useCountStepTable = ({
       };
       // Table consists of text fields, one numerical field for quantity counted,
       // select field for bin locations and one date picker for the expiration date.
-      const type = getFieldType(id);
-      const Component = getFieldComponent(id);
-      const fieldProps = getFieldProps(id);
-      const showTooltip = id === 'binLocation';
+      const type = getFieldType(columnPath);
+      const Component = getFieldComponent(columnPath);
+      const fieldProps = getFieldProps(columnPath);
+      const showTooltip = columnPath === cycleCountColumn.BIN_LOCATION;
+
+      // Columns allowed for focus in new rows
+      const newRowFocusColumns = [
+        cycleCountColumn.LOT_NUMBER,
+        cycleCountColumn.EXPIRATION_DATE,
+        cycleCountColumn.QUANTITY_COUNTED,
+        cycleCountColumn.COMMENT,
+      ];
```

Comment:
I think it's cells, maybe it should be named just `focusableCells` or `newRowFocusableCells`?